src/components/Footer.tsx

```diff
@@ -118,7 +122,8 @@ export class Footer extends React.PureComponent<IFooterProps, IFooterState> {
           className={styles.restart}
         />
         <a
-          href="https://www.datacamp.com/?utm_source=datacamp_light&utm_campaign=powered_by_datacamp"
+          href={`https://www.datacamp.com/?utm_source=${this.props
+            .utmSource}&utm_campaign=${this.props.utmCampaign}`}
```

Comment:
💭 this is probably caused by the linter, but the way this line is split really affects readability IMO. maybe we can move this somewhere else, and also even use the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) and [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interfaces?

Reply:
URL and URLSearchParams not used since current TS version does not support it